Makefile

```diff
@@ -28,6 +28,11 @@ endif
 GOTAGSLIST  += osusergo netgo static_build
 GOBUILDMODE := -buildmode pie
 endif
+ifeq ($(ARCH), arm)
```

Comment:
could you make this more specific, so that raspbian based builds would not have these variables set ?
i.e. I'd like to see if we could transition to your docker build without modifying the existing system.

Reply:
Sure, adding that now 👍

Reply:
I attempted to resolve this